scripts/scripts.js

```diff
@@ -530,10 +533,52 @@ async function loadEager(doc) {
   }
 }
 
+function setUpSoftNavigation() {
+  document.body.addEventListener('click', async (e) => {
+    const link = e.target.closest('a');
+    if (link && getMetadata('template') === 'guides') {
+      const { href } = link;
+      const hrefURL = new URL(href);
+      if (hrefURL.origin === window.location.origin) {
+        e.preventDefault();
+        try {
+          const resp = await fetch(href);
+          const html = await resp.text();
+          const dom = new DOMParser().parseFromString(html, 'text/html');
+          const main = dom.querySelector('main');
+          const template = dom.querySelector('meta[name="template"]');
+          if (template && template.getAttribute('content') === 'guides') {
+            await decorateMain(main);
+            await loadBlocks(main);
+            const currentMain = document.querySelector('main');
+            const children = [...currentMain.children].slice(2);
+            children.forEach((child) => child.remove());
```

Comment:
```suggestion
            sampleRUM('leave');
            children.forEach((child) => child.remove());
```

This will record a `leave` checkpoint as we ale dropping the content of the page now. @chicharr – I'm going straight to the checkpoint instead of calling `sampleRUM.leave()` which has additional guardrails